### PR TITLE
Update test1

### PR DESCRIPTION
**Pull request by Rabiprd07**
defect 1 fixed

### CODE REVIEW

File: test1 [hunk 1]
```diff
@@ -1,2 +1,3 @@
 first file for this organization
 This is my first change
+defect 1 is fixed
```
Comment (rabituktuk) [inline test1]: looks good

Comment (rabituktuk) [reply test1]: but gives comments

**Review by rabituktuk — CHANGES_REQUESTED** [review]
gives comment for code

File: test1 (hunk 1 above)
Comment (Rabiprd07) [reply test1]: comment is given in code. please check once

File: test1 [hunk 2]
```diff
@@ -1,3 +1,3 @@
 first file for this organization
 This is my first change
-defect 1 is fixed
+defect 1 is fixed  // it is for testing purpose only
```
Comment (rabituktuk) [inline test1:3]: thanks for comments

**Review by rabituktuk — APPROVED** [review]
all goods

**Comment by Rabiprd07** [comment]
verified

**Pull request merged**